### ISSUE TITLE
Add support for Mac OS X

### ISSUE THREAD

**Issue by bryongloden** [cryptolok/CryptoTrooper#1]
Mac OS X is a unix-based OS (Unix -> BSD -> NeXTSTEP -> Mac OS X) and doesn't have apt-get. There is a package manager called Homebrew that is used instead 😁

**Comment by cryptolok** [comment]
Em...well ,Mrs CISSP, let me tell you something.
In the late 2015 there was the first know Linux ransomware but since, its creators aren't knew shit about crypto, they generated the symmetric encryption key based on the file's timestapm, so their malware was cracked pretty quickly.
Now, as you might know, most of the servers are Linux-based, and the majority of them are Debian-based (including Ubuntu), hence the Linux ransomware is a huge potential threat to the industry, at least In My Hackish Opinion.
So I tried to show its potential and to make something fundamentally different from the others, but I'm not willing to destroy the whole industry, so I made it cryptographically broken, in order it could be cracked easily if went wild (although, I could make a strong mathematically enchased version and probably make a lot of money on it).
I don't see any reason to make a Windows version, not only because I hate this OS, but there's already a lot of market-level malware for it and Windows Servers are in minority (I hope they will).
Speaking about Mac, it's just a fancy home user OS as for me, it's true that, there isn't a lot of ransomware, but you can actually use mine without any package installation on any Unix (including BSD) through BASH/SH/KSH so, you don't really need a package installation.
In conclusion, I don't see any need for developing a specified version for a Mac OS X for now, however, feel free to fork my projcet and do whatever you want with it.
